exercises/numbers/isEven/isEven.js

```diff
@@ -15,7 +15,11 @@
  */
 function isEven(num) {
   // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 == 0 ) {
```

Comment:
The expression `num % 2 === 0` evaluates to `true` or `false`, so you can just do:

```javascript
return num % 2 === 0;
```

Be careful with `==` versus `===`.  For example, `5 == '5'` is `true`, but `5 === '5'` is `false`. With `==`, JavaScript will convert the data you give it to the same type (or try to). With `===`, if two items are of a different type, it will always yield `false`.

Something like:

```javascript
let num = 5;
let str = '5';

if (num === Number(str)) {
  // ...
}
```

make it clear what you intend for the code to do.  (`Number('5')` will convert the string `'5'` to the number `'5`.)